numba/parfors/array_analysis.py

```diff
@@ -1766,11 +1769,11 @@ def _analyze_op_call(self, scope, equiv_set, expr):
                 expr.args = args[1:]
             return result
 
-    def _analyze_op_call___builtin___len(self, scope, equiv_set, args, kws):
+    def _analyze_op_call___builtin___len(self, scope, equiv_set, loc, args, kws):
         # python 2 version of len()
```

Comment:
Py 2... dead?

Reply:
i'll go after all the `__builtin__` references

Reply:
they are removed